jsonpb/jsonpb_test.go

```diff
@@ -789,6 +789,18 @@ func TestUnmarshalJSONPBUnmarshaler(t *testing.T) {
 	if msg.rawJson != rawJson {
 		t.Errorf("message contents not set correctly after unmarshalling JSON: got %s, wanted %s", msg.rawJson, rawJson)
 	}
+
+	rawJson = `{"stringField":null}`
+	var ptrFieldMsg ptrFieldMessage
+	if err := Unmarshal(strings.NewReader(rawJson), &ptrFieldMsg); err != nil {
+		t.Errorf("an unexpected error occurred when parsing into JSONPBUnmarshaler: %v", err)
+	}
+	if ptrFieldMsg.StringField == nil {
```

Comment:
May be more precise to compare with expected value.  Use proto.Equal to compare.

Error message can be something like "unmarshal result StringField: got %v, want %v"

Reply:
Done